docs/storybook/src/components/MessageCenterField.stories.tsx

```diff
@@ -11,6 +11,7 @@ import {
   OutputMessagePriority,
 } from "@itwin/core-frontend";
 import { DropdownButton, MenuItem } from "@itwin/itwinui-react";
```

Comment:
I'd expect a toggle that enables `useStrataKit` preview feature in the existing story of `MessageCenterField` (no other changes).